src/Generator/AbstractTypeGenerator.php

```diff
@@ -145,6 +145,22 @@ protected function varExport($var, $default = null, array $compilerNames = [])
             case is_object($var):
                 return $default;
 
+            case is_string($var):
```

Comment:
I updated the Star Wars schema to have a description which would trigger this bug - or would you prefer a more involved test?